charts/tekton-pipelines/templates/buildpacks/trigger-templates/_buildpack.yaml

```diff
@@ -75,6 +75,10 @@ spec:
           value: "$(tt.params.buildpack_config_filename)"
         - name: project_config_filename
           value: "$(tt.params.project_config_filename)"
+        - name: project
```

Comment:
do you have a related tekton-apps change for the trigger bindings?

Reply:
We don't need this, because we already have these params in our trigger bindings by default